examples/tailwindcss/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@lynx-example/tailwindcss",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
```suggestion
```

If we are going to use `<Go />` with this example on the lynx-website, we need to publish it to NPM.